File: TemplateAPI/TemplateAPI/src/Domain/Entities/Audit.cs
namespace TemplateAPI.Domain.Entities;

public class Audit : BaseAuditableEntity
{
    public string UserId { get; set; } = string.Empty;
    public string Type { get; set; } = string.Empty;
    public string TableName { get; set; } = string.Empty;
    public DateTime AuditTime { get; set; }
    public string OldValues { get; set; } = string.Empty;
    public string NewValues { get; set; } = string.Empty;
    public string AffectedColumns { get; set; } = string.Empty;
    public string PrimaryKey { get; set; } = string.Empty;
}

File: angular-template/src/app/modules/shared/models/audit.ts
export interface Audit {
  userID: string;
  type: string;
  tableName: string;
  dateTime: Date;
  oldValues: string;
  newValues: string;
  affectedColumns: string;
  primaryKey: string;
  created: Date;
  createdBy: string;
  lastModified: Date;
  lastModifiedBy: string;
}
